src/main/java/org/myrobotlab/service/config/ArduinoConfig.java

```diff
@@ -1,8 +1,32 @@
 package org.myrobotlab.service.config;
 
+import java.util.LinkedHashMap;
+
 public class ArduinoConfig extends ServiceConfig {
+  
 
   public String port;
   public boolean connect;
+  public String serial; // name of serial service 
+  
+  @Override
+  public LinkedHashMap<String, ServiceConfig> getDefault(String name) {
```

Comment:
shoudln't this be getDefaultConfig on the Arduino class?   why is it on the ArduinoConfig ?